Classes/Information/Validatation/InformationValidator.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace FriendsOfTYPO3\Kickstarter\Information\Validatation;
+
+use ReflectionClass;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class InformationValidator
+{
+
+    public function validate(object $information): void
+    {
+        $rc = new ReflectionClass($information);
+        $errors = [];
+
+        foreach ($rc->getProperties() as $prop) {
+            $attr = $prop->getAttributes(UseValidator::class)[0] ?? null;
+            if (!$attr) {
+                continue;
+            }
+
+            /** @var UseValidator $meta */
+            $meta = $attr->newInstance();
+            $validator = GeneralUtility::makeInstance($meta->serviceId);
```

Comment:
Please create a new method for instance creation. That way you can define UseValidator as argument type and you have better control of return type. No need to add @var annotation anymore.